packages/api-gateway/src/drivers/http/adapters/messages/handlers.js

```diff
@@ -77,7 +77,27 @@ async function listChatMessages(req, reply) {
   return reply.code(200).send(paginatedResult);
 }
 
+/**
+ * Send new message from the requesting chat room
+ * @type {import('fastify').RouteHandler}
+ */
+async function saveNewMessage() {
+  // eslint-disable-next-line prefer-arrow-callback
+  this.io.on('connection', function (socket) {
```

Comment:
I think we don't need this

Reply:
So how are we going to get socket instance?